### ISSUE TITLE
Final Peer Review rt359

### ISSUE THREAD

**Issue by rt359** [mchao409/ORIE-4741-Class-Project#12]
This project aims to predict the number of bikes that will be rented from a particular station sin San Francisco. The goal is to help riders determine what are the best times to rent bikes and also for the bike company to decide how many buses to provide at certain tims and locations, as well as provide insight into how they could better serve their customers. 

Things I like: 
1) I like that the group imputes missing precipitation level columns instead of averaging or dropping those rows and uses a variety of loss functions in the process. I also like how the group dealt with corrupted zip code values. 
2) like that the group considers a decision tree model in addition to the linear regression models and analyzes at a variety of depths. 
3) The description of the group’s methodology in selecting different regularizers and loss functions is thorough and displays a clear understanding of the concepts discussed. 

Areas of improvement: 
1) It would have been useful to see more visuals, for example graphs from the regression modeling or a better way of presenting which variables had positive or negative coefficients (and the implications of these). 
2) It would have been useful to consider more implications of the model—for example identifying unusual demand pattern at specific locations or if there are unusual patterns in ridership around certain times of the year. The groups most notable insight at the moment is that precipitation has a negative coefficient, which seems intuitive. 
3) For next steps, it would have been useful to say why the group was considering exploring neural nets. Perhaps other next steps to explore would be incorporating other data sets or seeing how applicable the endings would be for a bike share system in a different city, for example.
